src/controllers/oidc/account/post.ts

```diff
@@ -23,15 +29,28 @@ export async function controller(req: Request, res: Response) {
     const account = await AccountService.get(session.accountId);
     const file = (req.files as any)?.profile?.[0] as Express.Multer.File;
     const body = { ...req.body };
+
     if (!account) throw new Error(ERROR_NO_ACCOUNT);
 
+    const isEmailChanged = req.body.email && account.email.toLowerCase() != req.body.email.toLowerCase();
+
     if (file) {
         const profileImg = await UploadProxy.post(file);
         body['profileImg'] = profileImg;
     }
 
     await AccountService.update(account, body);
 
+    if (isEmailChanged) {
+        account.isEmailVerified = false;
+        account.verifyEmailToken = createRandomToken();
+        account.verifyEmailTokenExpires = DURATION_TWENTYFOUR_HOURS;
+        await account.save();
+
+        // SEND VERIFICATION REQUEST FOR THE NEW EMAIL
```

Comment:
@valeriagrazzini Imo the method name is already quite explanatory and a comment like this seems a bit redundant. I've seen you do this in quite some places so I'm curious what the reasoning behind it is. Could you elaborate?